app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/provision_workflow.rb

```diff
@@ -180,6 +180,10 @@ def validate_ip_address(_field, _values, _dlg, _fld, value)
     return _('IP-address field has to be either blank or a valid IPv4 address') unless valid
   end
 
+  def validate_placement_group(_filed, _value, _dlg, _fld, value)
+    String.new(value.strip)
```

Comment:
Hey @Kuldip-Nanda what issue are you trying to workaround here?  if `value` is `nil` then `.strip` will fail, but `.strip` is going to return a string

Reply:
Do you need to return an empty string when value is nil? If so, maybe something like this or similar:

```
def validate_placement_group(_filed, _value, _dlg, _fld, value)
  value&.strip || ''
end
```

Reply:
I have already given a default value of the empty string to placement_group, but this is better.

Reply:
Oh I see now, you've set the defaukt in the YAML.

Reply:
Unresolving this comment since it is still applicable